app/views/dashboard/index.html.erb

```diff
@@ -130,13 +130,15 @@
               <%= new_button_to new_donation_path, {text: "New Donation"} %>
               <h3 class="text-center"><%= total_received_donations %> items
                 received <%= @selected_date_range_label %></h3>
-              <h3 class="text-center">$<%= total_received_money_donations %>
-                raised  <%= @selected_date_range_label %></h3>
-              <div class="box-body">
-                <h4>Recent donations</h4>
-                <%= render partial: "donation", collection: @recent_donations, as: :donation %>
+              <% if total_received_money_donations > 0 %>
```

Comment:
Bumped this down into an if because `dollar_value` didn't return anything if the value was zero.